python/packaging.bzl

```diff
@@ -134,6 +140,8 @@ def _py_wheel_impl(ctx):
     args.add("--abi", ctx.attr.abi)
     args.add("--platform", ctx.attr.platform)
     args.add("--out", outfile.path)
+    if ctx.attr.stamp:
+        args.add("--stamp_info", ctx.info_file.path)
```

Comment:
Could this be `version_info` instead? I don't think the use of stamping should always trigger rebuilds and I feel like if folks want stable info, then a `version_file` attribute should probably be added allowing them to resolve it themselves.

Reply:
I could switch it to use version_file instead.  My current scenario required it to use the stable versions but if the volatile versions would be better for the general public and not cause it to rebuild then I'm fine with changing it.

Reply:
The only thing I care about is being able to stamp targets without triggering rebuilds. If stable status is required, then I'd rather have a utility to generate it separately from the wheel and somehow wire that up (eg, some `version_file` attribute which would contain the version to use).